pkg/trace/remoteconfighandler/remote_config_handler.go

```diff
@@ -77,7 +78,7 @@ func (h *RemoteConfigHandler) onUpdate(update map[string]state.APMSamplingConfig
 		}
 	}
 
-	log.Debugf("updating samplers with remote configuration: %v", samplerconfigPayload)
+	log.Debugf("updating samplers with remote configuration: %v", spew.Sdump(samplerconfigPayload))
```

Comment:
```suggestion
	log.Debugf("updating samplers with remote configuration: %s", spew.Sdump(samplerconfigPayload))
```
It looks like Sdump returns a string

Reply:
This is equivalent apparently, so unless you have a reason to prefer %s can I leave it like that? I personally use %v everywhere and it seems to be working fine

![image](https://user-images.githubusercontent.com/103181765/206485446-2aef81ef-e2b5-4c4d-a907-f46bf406f6e4.png)